components/clinical/flag-detail/src/index.tsx

```diff
@@ -61,19 +99,32 @@ const FlagDetail: FC<Props> = ({ flag, controls = [], viewType = DetailViewType.
 
   return (
     <CollapsibleDetailCollection viewType={viewType}>
-      <StringDetail term="Code" description={codeableConceptCodeSummary(flag?.code)} />
-      <CodeableConceptDetail term="Name" concept={flag?.code} />
-      <StringDetail term="Status" description={flag.status.toString()} />
-      <CodeableConceptDetail term="Category" concept={flag?.category} />
+      <TopSection>
+        <CodeableConceptDetail term="Code" concept={flag?.code} displayCode />
+        <CodeableConceptDetail term="Name" concept={flag?.code} />
+      </TopSection>
+      <TopSection>
+        <StringDetail term="Status" description={flag.status.toString()} />
+        <CodeableConceptDetail term="Category" concept={flag?.category} />
+      </TopSection>
       <PeriodDetail period={flag?.period} />
       <NarrativeDetail term="Description" narrative={flag?.text} />
       <StringDetail
         term="Advice"
         description={getStringExtension(flag?.extension, 'https://leedsth.nhs.uk/alert/advice')}
         parse={false}
       />
+      <AuthorSection>
+        <div>
+          <ResourceReferenceDetail term="Author" resourceReference={flag?.author} />
+          <DatetimeDetail term="Authored On" datetime={flag?.createdOn} />
+        </div>
+        <div>
+          <ResourceReferenceDetail term="Amended By" resourceReference={flag?.amendedBy} showIfEmpty={false} />
+          <DatetimeDetail term="Amended On" datetime={flag?.amendedOn} />
```

Comment:
Are `Amended By` and `Amended On` always shown? It sounds like they might sometimes not have values. In that case, we should show a `-` or remove them from the UI.

Reply:
when date is not present it dont show and setting showIfEmpty to false ensures that field is not visible when empty